src/components/tasks/taskCard.tsx

```diff
@@ -2,19 +2,28 @@ import React from 'react';
 import style from './taskCard.module.sass'
```

Comment:
This currently not in use because of it's inefficient design, `taskStrip` component is being used rather.